api/src/middlewares/services/UserService/personalisation/personalisationBuilder/userPostalConfirmation/index.ts

```diff
@@ -18,18 +19,16 @@ export function buildUserPostalConfirmationPersonalisation(answers: AnswersHashM
   const country = answers["country"] as string;
   const post = answers["post"] as string;
   const previousMarriage = answers.maritalStatus && answers.maritalStatus !== "Never married";
-
-  const countryIsSpain = country === "Spain";
-  const countryIsItaly = country === "Italy";
-
-  const livesInCountry = answers.livesInCountry === true;
-
-  // Italy and Spain are the only countries that requires the partner's proof of end of marriage doc
-  const partnerHasPreviousMarriage = answers.partnerMaritalStatus && answers.partnerMaritalStatus !== "Never married";
-  const italySpainPartnerPreviousMarriageDocNeeded = (countryIsItaly || countryIsSpain) && partnerHasPreviousMarriage;
-
-  // For Croatia, there is an additional question asking if the user needs a certificate of custom law. If the answer is yes, they will need to provide this with their postal application
-  const croatiaCertNeeded = answers.certRequired === true;
+  const getAdditionalPersonalisations = postalPersonalisationsByCountry[country];
```

Comment:
maybe `getAdditionalCountryPersonalisation` (or context - since we're already inside `getUserPostalConfirmationAdditionalContext`?)?

Reply:
I'll go with `getAdditionalCountryPersonalisation` because I think `getAdditionalCountryContext` might get it confused with the `additionalContexts` being pulled in above